daily_post.py

```diff
@@ -53,13 +83,17 @@ def get_weather_emoji(period: Dict[str, Any]) -> Optional[str]:
     if re.fullmatch(rainy_regex, short_forecast):
         return Emojis.Rain
     elif re.fullmatch(cloudy_regex, short_forecast):
-        return Emojis.Cloudy
+        return Emojis.Cloud
     elif re.fullmatch(partly_cloudy_regex, short_forecast):
-        return Emojis.PartlyCloudy
+        if night:
+            date = _convert_time_str_to_datetime(period['startTime']).date()
+            return get_moon_phase(date) + Emojis.Cloud
```

Comment:
As it currently stands, this would bail out if there were an error yeah? Can we make it so it fails gracefully and still posts something?

Reply:
It'll post something, just won't post any emojis (the forecasts will still go through). I'm having it bail out on purpose so that if there is an issue, it'll get reported to me via PM.

See the try/except in get_weather.